run/test/specs/utils/check_colour.ts

```diff
@@ -23,3 +24,28 @@ export async function parseDataImage(base64: string) {
   // console.info("Middle x:", middleX, "middleY:", middleY, "width:", width);
   return pixelColor;
 }
+
+// Function to compare two colors within a specified CIEDE2000 tolerance
+export function compareColors(
+  hex1: string,
+  hex2: string,
+  tolerance: number = 2.3 // looks-same default value which is "enough for most cases"
+): boolean {
+  // looks-same expects colors as RGB objects but parseDataImage outputs hex
+  function hexToRgbObject(hex: string): { R: number; G: number; B: number } {
+    const bigint = parseInt(hex.replace('#', ''), 16);
```

Comment:
[BigInt](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt) is a reserved keyword in JS. It's a bit weird to name something that close. Maybe rename it to `asInt` or something else?